src/twiliosim.cr

```diff
@@ -31,28 +30,81 @@ class Twiliosim::Server
       response = {"sid" => UUID.random().to_s()}
       response.to_json(context.response)
     when %r(.+/Calls.*)
-      params = HTTP::Params.parse(request.body.not_nil!.gets_to_end)
-      @verboice_url = params["Url"]
+      account_sid = (/\/Accounts\/(.+)\/Calls.*/.match(request.path).try &.[1])
```

Comment:
Maybe the regex in the previous line can match the account sid directly. `$1` `$2` are available to get the captured group